convert2rhel/systeminfo.py

```diff
@@ -204,7 +205,13 @@ def _get_repofile_pkgs(self):
         return self._get_cfg_opt("repofile_pkgs").split()
 
     def _get_releasever(self):
-        """Get the release version to be passed to yum through --releasever.
+        """Releasever is used to referance the Version of Oracle Linux or Centos
+        to a variable to be converted to when going through the conversion that will
+        be put in a repo baseurl. By default the RELEASE_VER_MAPPING is used but if it
+        the var can’t be found then the releasever will be used, to find the version
+        specific config files in the convert2rhel/convert2rhel/data .
+
+        Get the release version to be passed to yum through --releasever.
```

Comment:
* Similar to a git commit message, docstrings usually have a one line summary at the top, then a blank line, and then a longer explanation.  So I'd put Line 214 back at the top.

* I think the releasever is actually the version of RHEL that we're converting too rather than the release of Oracle or CentOS that we're converting from.

* The language around whether RELEASE_VER_MAPPING or the config file takes precedence is a bit confusing.  I think what it should say is:
> By default, the value is found by mapping from the current distro's version to a compatible version of RHEL via the RELEASE_VER_MAPPING.  This can be overridden by the user by specifying it in the config file.  The version specific config files are located in convert2rhel/convert2rhel/data.

* Note that some of that was taken from the existing docstring (the two lines after this comment).  It's probably best to merge the information together like this rather than having two separate sections that say almost the same thing.

Reply:
Ok I have pushed a edited version of the docstring to take in account this comment!